components/sections/JoinCommunity.tsx

```diff
@@ -36,15 +36,14 @@ const JoinCommunity = () => {
             </Text>
             <VStack maxW={{ base: "md", md: "full" }} spacing="6" alignItems="start">
               <Text fontSize={{ base: "md", lg: "base" }} textAlign="left" color="brand.grey-40">
-                Shardeum is a global community. Anyone can join us in the journey to onboard
-                billions of people to Web 3. Don&apos;t know how to code? Or not sure where to
-                start? Don&apos;t worry. You can contribute as a content creator, community manager,
-                language translator, developer and you name it.
+                {
```

Comment:
I don't see any expressions so `{` can be removed.

Reply:
Ok